web/src/rest/publish.ts

```diff
@@ -128,24 +128,24 @@ const publishRest = new Vue({
       const version = (await this.versions(identifier, { page: count, page_size: 1 })).results[0];
       return girderize(version);
     },
-    async dandisets(params) {
+    async dandisets(params: any = undefined) {
       return client.get('dandisets/', { params });
     },
-    async createDandiset(name, description) {
+    async createDandiset(name: string, description: string) {
       const metadata = { name, description };
       return client.post('dandisets/', { name, metadata });
     },
-    async owners(identifier) {
+    async owners(identifier: string) {
       return client.get(`dandisets/${identifier}/users/`);
     },
-    async setOwners(identifier, owners) {
+    async setOwners(identifier: string, owners: [any]) {
       return client.put(`dandisets/${identifier}/users/`, owners);
     },
-    async searchUsers(username) {
+    async searchUsers(username: string) {
       const { data } = await client.get('users/search/?', { params: { username } });
       return data;
     },
-    assetDownloadURI(asset) {
+    assetDownloadURI(asset: any) {
```

Comment:
Again, this might be a good place to define the `Asset` type, along with the others. Since these type are pretty universal, they could probably go in `web/@types/` or something similar, and be imported from there.

Reply:
For the record I both mistyped and chose a wrong name for the types folder, as @dchiquit pointed out offline. I believe this will end up being `/web/src/types/`.